src/init.cpp

```diff
@@ -917,7 +917,7 @@ bool AppInitParameterInteraction(const ArgsManager& args)
 
     // Testnet3 deprecation warning
     if (chain == ChainType::TESTNET) {
-        LogInfo("Warning: Support for testnet3 is deprecated and will be removed in an upcoming release. Consider switching to testnet4.\n");
+        LogInfo("Warning: Support for testnet3 is deprecated and will be removed in an upcoming release (current target v30). Consider switching to testnet4.\n");
```

Comment:
Looks like `feature_config_args.py` needs to be updated to reflect the updated log.  Something like:

```diff
diff --git a/test/functional/feature_config_args.py b/test/functional/feature_config_args.py
index bb20e2baa85..201112ba0d5 100755
--- a/test/functional/feature_config_args.py
+++ b/test/functional/feature_config_args.py
@@ -378,7 +378,7 @@ class ConfArgsTest(BitcoinTestFramework):
 
     def test_testnet3_deprecation_msg(self):
         self.log.info("Test testnet3 deprecation warning")
-        t3_warning_log = "Warning: Support for testnet3 is deprecated and will be removed in an upcoming release. Consider switching to testnet4."
+        t3_warning_log = "Warning: Support for testnet3 is deprecated and will be removed in an upcoming release (current target v30). Consider switching to testnet4."
 
         def warning_msg(node, approx_size):
             return f'Warning: Disk space for "{node.datadir_path / node.chain / "blocks" }" may not accommodate the block files. Approximately {approx_size} GB of data will be stored in this directory.'
```

Reply:
Automatically fixed since I have removed the version reference.